src/workerd/api/streams/common.h

```diff
@@ -14,6 +14,11 @@
 #include <deque>
 #include <queue>
 
+#ifdef _MSC_VER
+#include <BaseTsd.h>
```

Comment:
How much stuff is in this header? Windows headers have a bad habit of ruining the global scope. Is there any way we can avoid it? Maybe we can `typedef long long ssize_t` instead?